lib/sendgrid_api/delivery_methods/sendgrid.rb

```diff
@@ -1,5 +1,6 @@
 begin
   require 'mail/check_delivery_params'
+  require 'simpleidn'
```

Comment:
This shouldn't be in a rescued block because it's not optional (it's a declared dependency). If it gets rescued then we'd get errors looking for the module constant at runtime.